### DISCUSSION TITLE
fasthtml-sessionnormalizer - a custom session middleware implementation

### DISCUSSION THREAD

**Discussion by palomino79** [AnswerDotAI/fasthtml#352] [Show and tell]
I have a project I've created to add a custom session middleware for FastHTML (https://github.com/palomino79/session-normalizer-middleware). This middleware is designed to aggressively serialize anything passed that you might want to persist for a user. The motivation behind this was this issue: https://github.com/AnswerDotAI/fasthtml/issues/226, in which a user wanted to persist a UUID, but which the underlying implementation of FastHTML doesn't actually know how to do. Additionally, if @jph00 or another maintainer wouldn't mind giving me permission to name my package that I want to distribute "fasthtml-sessionnormalizer" I'd really appreciate it. I don't want to publish anything containing the fasthtml name without relevant permissions, as doing so might imply endorsement by the maintainers for the code that I write/will write/have written. 

After either being permitted to use "fasthtml" in the project name or being asked to change it to something else, I'll flesh out the project's README to demonstrate usage and how to install prior to distributing the package.

**Comment by jph00** [comment]
Permission granted! Exciting project :D Let us know when it's released, and
we'll add a link to it from our docs.

**Reply by palomino79** [reply]
Thanks so much, Jeremy. I really appreciate all the kind words and encouragement.

**Reply by palomino79** [reply]
@jph00 The project has been released via PyPi. Relevant updates have been made to the project's README. Please let me know if you'd like me to update the FastHTML docs, or if that's something you or another team member typically handles. I know FastHTML is undergoing rapid development as a project, but I'll do my best to keep this middleware up to date with any relevant changes that might impact it. Thanks again for all the support.
